_posts/2022-12-09-aws-sigv4-support-for-clients.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title:  "AWS SigV4 support for OpenSearch clients"
+authors:
+  - hvamsi
+  - theotr
+  - xtansia
+  - wbeckler
+  - monicank
+date: 2022-12-09
+categories:
+  - feature
+
+---
+
+## Introduction
+
+OpenSearch Clients now support the ability to sign requests using [AWS Signature V4](https://docs.aws.amazon.com/general/latest/gr/sigv4_signing.html). This has been a community request since we forked clients from Elasticsearch, and we’re happy to announce that we have completed work across all clients, in collaboration with external contributors. Signing requests using native clients has been a big requirement for accessing the Amazon OpenSearch Service on AWS using fine grained access controls. Having SigV4 support natively in clients avoids using cURL requests or other workarounds.
```

Comment:
```suggestion
OpenSearch Clients now support the ability to sign requests using [AWS Signature V4](https://docs.aws.amazon.com/general/latest/gr/sigv4_signing.html). This has been a community request since we forked clients from Elasticsearch, and we’re happy to announce that we have completed work across all clients, in collaboration with external contributors. Signing requests using native clients has been an essential requirement for accessing the Amazon OpenSearch Service on AWS using fine grained access controls. Having SigV4 support natively in clients avoids using cURL requests and other workarounds.
```